### PR TITLE
Added sample consent letter + updated broken link

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#8039 and /bcgov/entity#8049

*Description of changes:*
- fixed Build Name URL
- fixed typos/cleanup
- added Sample Consent Letter block
- updated app version

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/lower-info-area/lower-container.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
                   <v-img src="@/assets/images/lc-tm-search-icon.png"
                          height="60"
                          width="60"
-                         comstrain />
```
Comment (severinbeauvais) [inline src/components/lower-info-area/lower-container.vue:23]: I don't see a display difference here, but anyway, it's fixed :)

File: src/components/lower-info-area/lower-container.vue [hunk 2]
```diff
@@ -135,10 +137,6 @@ export default class LowerContainer extends Vue {
 <style lang="scss" scoped>
 @import "@/assets/scss/theme.scss";
 
-li {
-  font-size: 1rem;
```
Comment (severinbeauvais) [inline src/components/lower-info-area/lower-container.vue:139]: Not needed. The link item inherits this from the container.

**Review by eve-git — APPROVED** [review]
Looks good to me.

**Review by andrepestana-aot — APPROVED** [review]
LGTM

**Pull request merged**